### PR TITLE
Refactor to match new GraphQL schema

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Refactor components from `Header`and `OrderInfo` so that they don't use `@vtex/delivery-packages` any more, since all necessary splitting of items into delivery packages in now handled in `order-placed-graphql` resolvers.
Also, remove dependencies `store-graphql` and `@vtex/delivery-packages`.

#### What problem is this solving?
There was some code repetition and lots of logic that went into splitting items into delivery packages being done in the front-end.

#### How should this be manually tested?
`cd react && yarn test`

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.
- [x] Refactor

### CODE REVIEW

File: react/__tests__/Header.test.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 
 import Header from '../components/Header'
-import { orderGroupQuery as bankInvoiceNoDueDate } from '../mocks/bankInvoice'
 import { orderGroupQuery as bankInvoiceDueDate } from '../mocks/bankInvoiceLoggedIn'
+import { orderGroupQuery as bankInvoiceNoDueDate } from '../mocks/bankInvoiceNumberLoggedIn'
 import { orderGroupQuery as oneDelivery } from '../mocks/oneDeliverySimple'
 import { orderGroupQuery as onePickup } from '../mocks/onePickupSimple'
-import { orderGroupQuery as takeAwayOnly } from '../mocks/oneTakeAway'
+// import { orderGroupQuery as takeAwayOnly } from '../mocks/oneTakeAway'
```
Comment (nandoacoelho) [inline react/__tests__/Header.test.tsx]: Please avoid commented code

File: react/__tests__/Header.test.tsx [hunk 2]
```diff
@@ -26,35 +26,35 @@ describe('Confirmation messages', () => {
...
-        data={takeAwayOnly.orderGroup}
-        profile={takeAwayOnly.orderGroup[0].clientProfileData}
-        inStore
-      />
-    )
-
-    const newOrder = getByText('Start new order')
-    expect(newOrder.textContent).toBeDefined()
-  })
+  // it('should render "Start new order" button if inStore', () => {
```
Comment (nandoacoelho) [inline react/__tests__/Header.test.tsx]: ditto

File: react/__tests__/Header.test.tsx [hunk 3]
```diff
@@ -173,24 +173,24 @@ describe('Warnings', () => {
...
   })
 
-  it('should not render Warnings if that are only take away items', () => {
-    const { queryByTestId } = render(
-      <Header
-        data={takeAwayOnly.orderGroup}
-        profile={takeAwayOnly.orderGroup[0].clientProfileData}
-      />
-    )
+  // it('should not render Warnings if that are only take away items', () => {
```
Comment (nandoacoelho) [inline react/__tests__/Header.test.tsx]: ditto

File: react/__tests__/OrderInfo.test.tsx [hunk 4]
```diff
@@ -2,17 +2,18 @@ import React from 'react'
 
 import OrderInfo from '../components/OrderInfo'
 import OrderHeader from '../components/OrderInfo/OrderHeader'
-import { orderGroupQuery as multipleDeliveries } from '../mocks/bankInvoice'
+import { orderGroupQuery as bankInvoicePayment } from '../mocks/bankInvoiceLoggedIn'
 import { orderGroupQuery as oneDelivery } from '../mocks/oneDeliverySimple'
 import { orderGroupQuery as onePickUp } from '../mocks/onePickupSimple'
-import { orderGroupQuery as takeAwayOnly } from '../mocks/oneTakeAway'
+// import { orderGroupQuery as takeAwayOnly } from '../mocks/oneTakeAway'
```
Comment (nandoacoelho) [inline react/__tests__/OrderInfo.test.tsx]: ditto

File: react/__tests__/OrderInfo.test.tsx [hunk 5]
```diff
@@ -33,25 +34,25 @@ describe('OrderHeader', () => {
...
-  it('should render different buttons for take away orders', () => {
-    const orderInfo = getOrderFromOrderGroup(takeAwayOnly.orderGroup, 0)
-    const { getByText } = render(
-      <OrderHeader
-        orderInfo={orderInfo}
-        runtime={{ account: 'vtexgame1' }}
-        takeaway
-      />
-    )
+  // it('should render different buttons for take away orders', () => {
```
Comment (nandoacoelho) [inline react/__tests__/OrderInfo.test.tsx]: ditto

File: react/__tests__/Header.test.tsx (hunk 1 above)
Comment (victorhmp) [reply react/__tests__/Header.test.tsx]: Removed :)

File: react/__tests__/Header.test.tsx (hunk 2 above)
Comment (victorhmp) [reply react/__tests__/Header.test.tsx]: Removed

File: react/__tests__/Header.test.tsx (hunk 3 above)
Comment (victorhmp) [reply react/__tests__/Header.test.tsx]: Removed

File: react/__tests__/OrderInfo.test.tsx (hunk 4 above)
Comment (victorhmp) [reply react/__tests__/OrderInfo.test.tsx]: Removed

File: react/__tests__/OrderInfo.test.tsx (hunk 5 above)
Comment (victorhmp) [reply react/__tests__/OrderInfo.test.tsx]: Removed

**Pull request merged**